ntbs-integration-tests/NtbsWebApplicationFactory.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Linq;
+using EFAuditer;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using ntbs_integration_tests.Helpers;
+using ntbs_service.Models;
+
+namespace ntbs_integration_tests
+{
+    public class NtbsWebApplicationFactory<TStartup> : WebApplicationFactory<TStartup> where TStartup : class
+    {
+        protected override void ConfigureWebHost(IWebHostBuilder builder)
+        {
+            builder.ConfigureServices(services =>
+            {
+                var serviceProvider = new ServiceCollection()
+                    .AddEntityFrameworkInMemoryDatabase()
+                    .BuildServiceProvider();
+
+                services.AddDbContext<NtbsContext>(options => 
+                {
+                    options.UseInMemoryDatabase("Ntbs_Test_Db");

```

Comment:
This actually works really well and is pretty fast to spin up!

Reply:
great! that's some happy news :D